client/components/support-info/index.jsx

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+
+class SupportInfo extends Component {
+	static propTypes = {
+		text: PropTypes.string,
+		link: PropTypes.string,
+		privacyLink: PropTypes.oneOfType( [ PropTypes.string, PropTypes.bool ] ),
```

Comment:
Is it necessary to introduce the boolean usage here? Is it enough to just have an empty string by default, and check for empty below?